view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -56,19 +58,120 @@ define(
                 return this;
             },
 
+            /**
+             * Format Price
+             * 
+             * @param {float} amount - Amount to be formatted
+             * @return {string}
+             */
+            getFormattedAmount: function (amount) {
+                return priceUtils.formatPrice(amount, quote.getPriceFormat());
+            },
+
+            /**
+             * Get Installment minimum
+             * this function respects info from: https://www.omise.co/installment-payment
+             *
+             * NOTE: in the future this function should return data from capabilities object.
+             *
+             * @param {string} id - Bank ID
+             * @return {integer}
+             */
+            getInstallmentMinimum: function (id) {
+                switch (id) {
+                    case 'kbank': return 500;
+                    case 'bbl': return 500;
+                    case 'bay': return 300;
+                    case 'first_choice': return 300;
+                    case 'ktc': return 300;
+                }
+            },
```

Comment:
<img width="932" alt="Screen Shot 2562-05-28 at 05 52 06" src="https://user-images.githubusercontent.com/2154669/58441519-d1258b00-810c-11e9-99a4-69c7b6c3e1df.png">

Not to bring this topic back, but if you are talking about its readability, I think the right screen is way more cleaner 😂 
(just share opinion, not asking for change)

Reply:
@jacstn Thanks for re-opened this comment again. I'll mark it as resolved now 👍 👍